frontend/app/module/polling_stations/page/PollingStationListPage.tsx

```diff
@@ -0,0 +1,50 @@
+import { PollingStationType, usePollingStationList } from "@kiesraad/api";
+import { IconChevronRight } from "@kiesraad/icon";
+
+const labelForPollingStationType: { [K in PollingStationType]: string } = {
+  FixedLocation: "Vaste locatie",
+  Special: "Bijzonder",
+  Mobile: "Mobiel",
+};
+
+export function PollingStationListPage() {
+  const { pollingStations } = usePollingStationList();
+
+  return (
+    <>
+      <header>
+        <section>
+          <h1>Stembureaus</h1>
+        </section>
+      </header>
+      <main>
+        <article>
+          <table id="polling_stations" className="polling_stations_table">
+            <thead>
+              <tr>
+                <th>Nummer</th>
```

Comment:
```suggestion
                <th className="number">Nummer</th>
```

Reply:
Added in https://github.com/kiesraad/abacus/pull/453/commits/26acb7b4e4300c1ec30d77b80e48b55669a0660d